<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcWindow</class>
 <widget class="QMainWindow" name="CalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minha Calculadora</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parcela 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parcela 2:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditResult">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>471</width>
      <height>64</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado:</string>
    </property>
    <property name="buddy">
     <cstring>textEditResult</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>250</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;C A L C U L A D O R A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="execButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Operações</string>
    </property>
    <widget class="QRadioButton" name="radioButtonMult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Multipilicação</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSoma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Soma</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSub">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Subtração</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDiv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Divisão</string>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="input2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <zorder>input2</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>textEditResult</zorder>
   <zorder>label_4</zorder>
   <zorder>exitButton</zorder>
   <zorder>label</zorder>
   <zorder>execButton</zorder>
   <zorder>groupBox</zorder>
   <zorder>input1</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRelatorio">
    <property name="title">
     <string>Relatorio</string>
    </property>
    <addaction name="actionByUser"/>
    <addaction name="actionAllUsers"/>
   </widget>
   <addaction name="menuRelatorio"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionByUser">
   <property name="text">
    <string>Por Usuário</string>
   </property>
  </action>
  <action name="actionAllUsers">
   <property name="text">
    <string>Todos os Usuários</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>textEditResult</tabstop>
  <tabstop>radioButtonSoma</tabstop>
  <tabstop>radioButtonSub</tabstop>
  <tabstop>radioButtonMult</tabstop>
  <tabstop>radioButtonDiv</tabstop>
  <tabstop>execButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>CalcWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>textEditResult</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input1</sender>
   <signal>valueChanged(double)</signal>
   <receiver>textEditResult</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
